src/ctap/crypto_wrapper.rs

```diff
@@ -214,6 +196,15 @@ fn ecdsa_key_from_seed(
     Ok(ecdsa::SecKey::from_bytes(&ecdsa_bytes).unwrap())
 }
 
+impl From<&PrivateKey> for cbor::Value {
+    fn from(private_key: &PrivateKey) -> Self {
+        cbor_array![
+            cbor_int!(private_key.signature_algorithm() as i64),
+            cbor_bytes!(private_key.to_bytes()),
+        ]
+    }
+}
+
 impl From<PrivateKey> for cbor::Value {
```

Comment:
Do we still need this version `from`? If we remove it, we don't have to worry about consistency. Doesn't look like a big difference in performance, since `to_bytes` takes a reference. If we want both, we probably want a test to check they are identical. Or one calls the other.

Reply:
Done